packages/dashboard/src/app/views/utils/getTemplateFilters.js

```diff
@@ -33,7 +33,7 @@ function getTemplateFilters(templates) {
   const colors = templatesArr
     .flatMap((t) => t.colors)
     .map((c) => ({
-      ...c,
+      label: c.family,
       value: c.label,
```

Comment:
might be nice to make these both the family:
```
label: c.family,
value: c.family,
```

I don't think we're even using the value, but just for safe keeping.

Reply:
yeah good point, was trying to be the least invasive - like keeping it tight though.